resources/views/users/connections.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layout')
+
+@section('title')
+    View your connection(s)!
+@stop
+
+@section('content')
+    You have {{count($connections)}} connection(s)!</br> </br>
+
+    @forelse($connections as $connection)
+        <div>
+            <?php $user = App\User::findOrFail($connection->owner_id) ?>
```

Comment:
If you really wanted to use PHP in a blade template use the `@php` directive instead.

But also don't use that directive and instead don't do this.

Reply:
fixed, same reason as above